src/v/security/jwt.h

```diff
@@ -96,27 +96,14 @@ time_point(json::Value const& doc, std::string_view field) {
 }
 
 template<string_viewable StringT = bytes>
-auto base64_url_decode(bytes_view sv) {
-    // TODO: Replace this with non-CryptoPP implementation
-    // TODO: https://github.com/redpanda-data/core-internal/issues/1132
-    CryptoPP::Base64URLDecoder decoder;
-
-    decoder.Put(sv.data(), sv.size());
-    decoder.MessageEnd();
-
-    StringT decoded;
-    if (auto size = decoder.MaxRetrievable(); size != 0) {
-        decoded.resize(size);
-        decoder.Get(
-          reinterpret_cast<CryptoPP::byte*>(decoded.data()), decoded.size());
-    }
-    return decoded;
+auto base64_url_decode(std::string_view sv) {
```

Comment:
question: are the template type and the `auto` deducted return type still needed?

Reply:
It may not, i'll check